dask/array/tests/test_fft.py

```diff
@@ -108,3 +113,41 @@ def test_fft_consistent_names():
     assert same_keys(fft(darr, 5), fft(darr, 5))
     assert same_keys(fft(darr2, 5, axis=0), fft(darr2, 5, axis=0))
     assert not same_keys(fft(darr, 5), fft(darr, 13))
+
+
+def test_wrap_numpy():
```

Comment:
> Can we refactor this to use `pytest.mark.parametrize`?

Probably to some extent. Though I don't think we can lump everything into one test. Please see this [note]( https://github.com/dask/dask/pull/2093#discussion_r106466514 ).